mlperf_logging/mllog/constants.py

```diff
@@ -95,6 +95,8 @@
 OPT_LR_ALT_WARMUP_FUNC = "opt_learning_rate_alt_warmup_func"
 OPT_LR_DECAY_BOUNDARY_EPOCHS = "opt_learning_rate_decay_boundary_epochs"
 OPT_LR_DECAY_BOUNDARY_STEPS = "opt_learning_rate_decay_boundary_steps"
+OPT_LR_DECAY_START_STEP = "opt_learning_rate_decay_start_step"
+OPT_LR_DECAY_NUM_STEPS = "opt_learning_rate_decay_steps"
```

Comment:
this is already there in `OPT_LR_DECAY_STEPS`

Reply:
sure, done